server/lib/schema/index.ts

```diff
@@ -2094,6 +2094,64 @@ const schema = new GraphQLSchema({
           },
         ),
       },
+      updateTreeExpertAssignments: {
```

Comment:
In the Mosaic codebase I'd prefer for the concept "assignment" to uniformly mean the assignment of a user to a workspace during an experiment. So I'd prefer we change the name of this mutation to something like `updateTreeExperts`.

Reply:
Good catch. I think it may be worthwhile to update `mosaic-api-scripts` as well to reduce confusion. Will talk to ben about that.

Reply:
I'm honestly not too worried about that. I think things are modular enough that I'm happy to defer to Ben R. and Mati about however they want to organize the scripts themselves.

Reply:
I do think this is a good point -- I had been using "assignment" to refer to "we assigned this expert to be on this tree via the API".

The new terminology I've been using (though haven't standardized everywhere, e.g. in mosaic-api-scripts) is "allocation" -- "we allocated this expert to be on this tree via the API"

Definitely taking suggestions for better terminology though :)

Reply:
I do think this is a good point -- I had been using "assignment" to refer to "we assigned this expert to be on this tree via the API".

I like "designate"/"designations" as @andrewschreiber suggested below, will plan to switch to that unless someone else wants to propose something different